statsmodels/genmod/tests/test_glm.py

```diff
@@ -204,7 +204,7 @@ def test_compare_discrete(self):
         score1 = res1.model.score(res1.params * 0.98)
         assert_allclose(score1, score_obs1.sum(0), atol=1e-20)
         score0 = res1.model.score(res1.params)
-        assert_allclose(score0, np.zeros(score_obs1.shape[1]), atol=1e-7)
+        assert_allclose(score0, np.zeros(score_obs1.shape[1]), atol=5e-7)
```

Comment:
Reason for this change?  In sm2 I had to change this from 1e-7 to 1.01e-7 because of a failure on Travis (that I couldn't reproduce locally).  If this is the same error/mismatch, then x5 may be more than necessary.

Reply:
sounds the same test failure
The test failure was just barely above 1e-7.

Usually I don't fine tune the tol if it is smaller than 1e-3 or 1e-4 to more than 1 or 5 as leading digit. If rtol, atol are close to noise in one machine/version, then there might be another machine/version with a bit more or different noise.
1e-7 or 5e-7 does not really make a meaningful difference.